server/src/test/java/io/unitycatalog/server/utils/TestUtils.java

```diff
@@ -27,6 +27,7 @@ public class TestUtils {
 
     public static final String CATALOG_NEW_NAME = "uc_newtestcatalog";
     public static final String CATALOG_NEW_COMMENT = "new test comment";
+    public static final String SPARK_CATALOG = "spark_catalog";
```

Comment:
I dont see a reason for the server side test class to have reference to something that is relevant only for Spark client stuff. So lets just keep the spark related tests self contained within spark module